runtime.md

```diff
@@ -15,9 +15,9 @@ This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
 * **`status`**: (string) is the runtime state of the container.
 The value MAY be one of:
-  * `created` : the container has been created but the user-specified code has not yet been executed
-  * `running` : the container has been created and the user-specified code is running
-  * `stopped` : the container has been created and the user-specified code has been executed but is no longer running
+  * `created` : the container process has neither exited nor executed the user-specified code
+  * `running` : the container process has executed the user-specified code but has not exited
```

Comment:
On Wed, Jul 13, 2016 at 09:43:58AM -0700, Daniel, Dao Quang Minh wrote:
> > @@ -15,9 +15,9 @@ This MUST be unique across all containers on this host.
> >  There is no requirement that it be unique across hosts.
> >  * **`status`**: (string) is the runtime state of the container.
> >  The value MAY be one of:
> > -  * `created` : the container has been created but the user-specified code has not yet been executed
> > -  * `running` : the container has been created and the user-specified code is running
> > -  * `stopped` : the container has been created and the user-specified code has been executed but is no longer running
> > +  * `created` : the container process has neither exited nor executed the user-specified code
> > +  * `running` : the container process has executed the user-specified code but has not exited
> 
> meaning of `user-specified code` vs `container process` here seems
> confusing to me.

This terminology issue is being hashed out in #466.  Maybe we should
handle that first?